anton.php

```diff
@@ -0,0 +1,14 @@
+<?php
+session_start();
+if ($_SESSION['username']=="anton") {
+	echo "Hello ".$_SESSION['username'];
```

Comment:
reuse [$users](https://github.com/Grisha86/form/pull/1/files#diff-ec8179f9475c380d2e1a7cb8261742aeR3) from form.php to display user name.